src/NLog/LayoutRenderers/StackTraceFormat.cs

```diff
@@ -52,5 +52,10 @@ public enum StackTraceFormat
         /// Detailed flat format (method signatures displayed in a single line).
         /// </summary>
         DetailedFlat,
+
+        /// <summary>
+        /// Raw format (multiline with source info - as returned by StackFrame.ToString() method).
+        /// </summary>
+        RawWithSource,
```

Comment:
I think fix "raw" is more clear :) 

So remove and we need to document the change in Nlog 4.4.7.